src/Traits/ModifierTrait.php

```diff
@@ -296,6 +296,8 @@ public function addYears(int $value): ChronosInterface
      */
     public function addYear(int $value = 1): ChronosInterface
     {
+        trigger_error('2.4 - addYear() is deprecated. Use addYears() instead.', E_USER_DEPRECATED);
```

Comment:
Let's us a similar format to cake core. `Since 2.4.0, ...`

Reply:
Sounds good.